packer/scripts/install-helm-images.rb

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env ruby
+
+require 'open3'
+require 'yaml'
+
+def lookup_nested(hash, key)
+    fragments = key.split('.')
+    frag = fragments[0]
+
+    unless hash.has_key?(frag)
+        return nil
+    end
+
+    current = hash[frag]
+    if fragments.length == 1
+        return current
+    end
+
+    key = key.sub(/^#{frag}\./, '')
+    lookup_nested(current, key)
+end
+
+def scrape_images(path)
+  values_yaml = File.join(path, 'values.yaml')
+  templates_dir = File.join(path, 'templates')
+
+  values = YAML.load(File.read(values_yaml))
+  templates = Dir.glob(File.join(templates_dir, '**/*'))
+
+  images = []
+
+  templates.each do |filename|
+      File.read(filename).each_line do |line|
+          # Check for a line that starts with image:
+          match = line.match(/^\s*-?\s*image:\s*"?(.*?)"?$/)
+          next if match.nil?
+
+          # Parse and replace any helm templates
+          images.push match[1].gsub(/{{\s*\.Values.([^}\s]+)\s*}}/) { lookup_nested(values, $1) }
```

Comment:
`/{{\s*\.Values.([^}\s]+)\s*}}/` should be the same as `/{{\s*\.Values.(.+?)\s*}}/`